shared/test/views_test.js

```diff
@@ -604,7 +604,7 @@ describe("loop.shared.views", function() {
       });
 
       it("should not throw if the element is not a video object", function() {
-        sinon.stub(view, "getDOMNode").returns({
+        sinon.stub(ReactDOM.findDOMNode(view)).returns({
```

Comment:
This change doesn't work as you're not actually stubbing anything - you're just giving sinon.stub an object, and not a second argument.

You need to change this (and the other 3 instances) to:

    sinon.stub(ReactDOM, "findDOMNode").returns(...